client/src/Clusters/AppOfTheWeek/AOTWStyle.css

```diff
@@ -0,0 +1,21 @@
+.aotw{
```

Comment:
Can you name this file AppOfTheWeekStyle.css instead. I had no idea what "aotw" was for the longest time